community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseBuilder.java

```diff
@@ -118,6 +119,16 @@ public GraphDatabaseBuilder setConfig( Map<String, String> config )
     }
 
     /**
+     * Retrieve the configured configuration values.
+     *
+     * @return the configured configuration values.
+     */
+    public Config config()
+    {
+        return new Config( config );
+    }
```

Comment:
@benbc if `Config` isn't part of the public API, is `GraphDatabaseBuilder` part of the public API? I have a feeling I have poor insight into what is public API these days.

What we need here is some way of reading the config, and `GraphDatabaseBuilder` did not provide any way of doing that before. I'm happy to take suggestions about other ways of achieving that.